content/en/references/logging.md

```diff
@@ -56,5 +56,5 @@ With Docker/Docker-Compose, you can run `docker ps` to get the container ID of t
 
 To view the logs via a user interface, you can use the following options:
 
-- [LocalStack Cockpit](https://docs.localstack.cloud/user-guide/tools/cockpit/)
+- [LocalStack Desktop App](https://docs.localstack.cloud/user-guide/tools/localstack-desktop/)
```

Comment:
I would keep the naming consistent so either "LocalStack Desktop" or "LocalStack Desktop App" across the docs